sigsci_site_manager/merge.py

```diff
@@ -0,0 +1,274 @@
+import json
+
+from sigsci_site_manager.backup import backups
+from sigsci_site_manager.util import filter_data, equal_rules
+
+
+def merge_rule_lists(api, data):
```

Comment:
Since all these commands require the API as a parameter, it would probably be better to create a client object that wraps the API instead of requiring the user to push this api object everywhere. This works for now though.

Reply:
This `api` object is a client object that wraps the API.